br-webui/views/network.liquid

```diff
@@ -195,6 +195,14 @@ function setCompanionEthernetIP() {
 	alert("New IP will take affect after reboot");
 }
 
+function setCompanionToDHCP() {
+
+	if (confirm("Are you sure you want to configure Companion as a DHCP server?")) {
+		socket.emit('set companion as dhcp server');
+	}
+	alert("For new changes to take effect, reboot the pi from the web interface and reconnect the ethernet cable to your computer after making necessary changes to your computer's network settings.");
```

Comment:
This is basically to ask if the user wants to change or not. until the user replies, the configuration does not change. only after the user says okay, it will configure companion as dhcp server. and yeah the alert message should only come when user says yes. not otherwise. you are right. I will change it.,